### ISSUE TITLE
Bug when deleting single-frame/multi-frame annotation with gap

### ISSUE THREAD

**Issue by mzur** [biigle/core#488]
An error occurs when the following steps are taken:

1. Create a multi-frame video annotation with two keyframes
2. Create a single-frame video annotation with a temporal gap to the first annotation
3. Link the multi-frame annotation to the single-frame annotation
4. Delete the keyframe of the single-frame annotation

The error occurs, the anotation is displayed wrongly and you have to repeatedly select different keyframes to get rid of the annotation. This should be fixed.

**Comment by mzur** [comment]
No, I don't see the warning. Can you provide steps to reporduce (or a video)?

Here is a video demonstrating the original issue:

https://github.com/biigle/core/assets/2457311/53723c5d-88e5-4b1a-aaba-02fe7e023649

**Comment by lehecht** [comment]
The error can be reproduced by:
1. create two whole frame annotations for two different labels
2. link them
The error should then appear on the console.
